### DISCUSSION TITLE
How datasets are organized using multimodal data to use MONAI's dataset classes

### DISCUSSION THREAD

**Discussion by LReion** [Project-MONAI/MONAI#6469] [Q&A] [answered]
**Please use MONAI's Discussions tab**
How datasets are organized using multimodal data to use MONAI's dataset classes

I have 3D data for T1, T2, how should I organize these two 3D data as two channel input models, and how the dataset organizes the 3D data of these two channels

**Comment by KumoLiu** [comment]
Hi @LReion, you could use `ConcatItemsd` to concat these two 3d data.
https://github.com/Project-MONAI/MONAI/blob/25c9c3951858bd60b4d02bcaca7fff9c4236bff7/monai/transforms/utility/dictionary.py#L1016

Hope it can help you, thanks!

**Reply by LReion** [reply]
I wanted to organize two 3D Ntifi file as two channel as a input, I need use MONAI dataset to load two 3d Ntifi file as two channels, but not use additional ways.

**Reply by KumoLiu** [reply]
I think you should have a compose transform containing maybe `LoadImage` and some other transforms that will send to Dataset, right?

**Reply by LReion** [reply]
Can LoadImage stitch keys like Label1 and Label2 in json files to stitch images?

**Reply by KumoLiu** [reply, marked as answer]
You could use `LoadImaged`.
```
transforms = Compose([
    LoadImaged(keys=["label1", "label2"], ensure_channel_first=True, image_only=True),
    ConcatItemsd(keys=["label1", "label2"], name="cat_label")
)
```

**Reply by LReion** [reply]
Thank for your answer.
